public/templates/header.php

```diff
@@ -35,7 +35,11 @@
       <li class="dropdown">
         <button class="dropbtn">Courses</button>
         <div class="dropdown-content">
-          <a href="course_list.php">Course List</a>
+          <?php
+          if (get_current_role() != "student") {?>
+            <a href="course_list.php">Course List</a>
+          <?php 
+          } ?>
```

Comment:
what if students navigate to that URL by hand? I think we need to do something there too

Reply:
or we can pretend that its fine 😂  that's cool too